### PR TITLE
auditd: migrate tile_map to map object

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

Migrates the map visualisation to from tile_map to map object.

(This PR is partially to test the waters — if it's fine I'll take on the remaining SEI migrations)

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] Please confirm that I have correctly adjusted the kibana version constraint.

## Related issues

- Relates #2617

### CODE REVIEW

File: packages/auditd/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "3.0.0"
+  changes:
+    - description: Migrate map visualisation from tile_map to map object
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/nnnn
```
Comment (efd6) [inline packages/auditd/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/3246
```

File: packages/auditd/kibana/dashboard/auditd-dfbb49f0-0a0f-11e7-8a62-2d05eaaac5cb.json [hunk 2]
```diff
@@ -27,11 +29,13 @@
                     "y": 0
                 },
                 "panelIndex": "1",
-                "panelRefName": "panel_0",
-                "version": "7.3.0"
+                "panelRefName": "panel_1",
+                "type": "visualization",
+                "version": "8.0.0-SNAPSHOT"
```
Comment (andrewkroh) [inline packages/auditd/kibana/dashboard/auditd-dfbb49f0-0a0f-11e7-8a62-2d05eaaac5cb.json:34]: Were you using stack version `8.0.0`? I'm surprised to see SNAPSHOT in the exported files.

**Review by andrewkroh — COMMENTED** [review]
LGTM. The only downside is that it appears we need to drop support for 7.x since the conversion process needs Kibana 8.0.

I think it would be good to get a review from a Kibana dashboard expert too. @flash1293, would you be able to look this over?

File: packages/auditd/kibana/dashboard/auditd-dfbb49f0-0a0f-11e7-8a62-2d05eaaac5cb.json (hunk 2 above)
Comment (efd6) [reply packages/auditd/kibana/dashboard/auditd-dfbb49f0-0a0f-11e7-8a62-2d05eaaac5cb.json:34]: I can redo that.

**Review by andrewkroh — APPROVED** [review]
LGTM, let's wait for another review before merging.

**Comment by efd6** [comment]
Yes. That was the plan, since I'll do the remainders as a single PR, so I want to make sure it only needs to be done once.

**Review by flash1293 — APPROVED** [review]
Took me a bit to set up my environment to test this, but ultimately I managed and this LGTM. The dashboard can be converted to by-value / Lens but this can also be handled in a separate PR.

**Comment by efd6** [comment]
@flash1293 Can you point me to docs for how to make that change? I will take that on after the maps migration.

**Comment by flash1293** [comment]
@efd6 Check out the [best practices doc](https://docs.google.com/document/d/1uyyFGx6xA5Kvl8c-ZdvXdvBGrHTylxU9F69TGqfzdmw/edit) - let me know in case some key information is missing, I'm definitely making assumptions about prior Kibana experience that might be incorrect for the targeted audience.

**Comment by efd6** [comment]
@flash1293 Thanks

**Pull request merged**